packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.types.ts

```diff
@@ -85,7 +85,7 @@ export interface IDetailsHeaderProps extends React.Props<DetailsHeaderBase> {
   viewport?: IViewport;
 
   /** Column reordering options */
-  columnReorderOptions?: IColumnReorderOptions | null;
+  columnReorderProps?: IColumnReorderProps;

```

Comment:
@ThomasMichon, currently ODSP-Next uses only DetailsList interface to enable dragdrop functionality in the list, and all these are internally used by DetailsList/DetailsHeader/DetailsColumn components. I have tested this with ODSP-Next and its working as expected. Do we still need to support both?, as it will impact the bundle size?